Makefile

```diff
@@ -9,21 +9,42 @@ ROOTFS_BUILDER := $(MK_DIR)/rootfs-builder/rootfs.sh
 INITRD_BUILDER := $(MK_DIR)/initrd-builder/initrd_builder.sh
 IMAGE_BUILDER  := $(MK_DIR)/image-builder/image_builder.sh
 
+DISTRO                :=
 AGENT_INIT            ?= no
-DISTRO                ?= centos
 ROOTFS_BUILD_DEST     := $(PWD)
 IMAGES_BUILD_DEST     := $(PWD)
-DISTRO_ROOTFS         := $(ROOTFS_BUILD_DEST)/$(DISTRO)_rootfs
-DISTRO_ROOTFS_MARKER  := $(ROOTFS_BUILD_DEST)/.$(DISTRO)_rootfs.done
-DISTRO_IMAGE          := $(IMAGES_BUILD_DEST)/kata-containers.img
-DISTRO_INITRD         := $(IMAGES_BUILD_DEST)/kata-containers-initrd.img
+TARGET_ROOTFS         := $(ROOTFS_BUILD_DEST)/$(DISTRO)_rootfs
+TARGET_ROOTFS_MARKER  := $(ROOTFS_BUILD_DEST)/.$(DISTRO)_rootfs.done
+TARGET_IMAGE          := $(IMAGES_BUILD_DEST)/kata-containers.img
+TARGET_INITRD         := $(IMAGES_BUILD_DEST)/kata-containers-initrd.img
 
 VERSION_FILE   := ./VERSION
 VERSION        := $(shell grep -v ^\# $(VERSION_FILE))
 COMMIT_NO      := $(shell git rev-parse HEAD 2> /dev/null || true)
 COMMIT         := $(if $(shell git status --porcelain --untracked-files=no),${COMMIT_NO}-dirty,${COMMIT_NO})
 VERSION_COMMIT := $(if $(COMMIT),$(VERSION)-$(COMMIT),$(VERSION))
 
+ifeq (,$(DISTRO))
```

Comment:
I'm not convinced about using the empty value to mean "use dracut" tbh.

I wonder if it would be "less surprising" if the user either had to set `DISTRO=dracut` (but that feels a little odd to me), or  `INITRAMFS_TYPE=dracut` (more sensible I think).

wdyt @jcvenegas?

Reply:
At the moment the default behavior when `DISTRO=...` is not passed is to build for centos. Is there a reason why this is the default?

I liked the fact that, when not specifying a distro, the image is based on the host's distro.

Reply:
@jcvenegas may remember why centos is the default.

Reply:
Ok, left `DISTRO := centos` and added `BUILD_METHOD` make var to control the build method ("distro" os "dracut").